contexts/RefetchContext.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { useRecoilValue } from 'recoil';
+import useSynthetixQueries from '@synthetixio/queries';
+
+import useGetFuturesOpenOrders from 'queries/futures/useGetFuturesOpenOrders';
+import useGetFuturesPositionForMarket from 'queries/futures/useGetFuturesPositionForMarket';
+import useGetFuturesMarketPositionHistory from 'queries/futures/useGetFuturesMarketPositionHistory';
+import { walletAddressState } from 'store/wallet';
+import useGetFuturesMarket from 'queries/futures/useGetFuturesMarket';
+import useGetFuturesPotentialTradeDetails from 'queries/futures/useGetFuturesPotentialTradeDetails';
+
+type RefetchType = 'modify-position' | 'new-order' | 'close-position' | 'margin-change';
+
+type RefetchContextType = {
+	handleRefetch: (refetchType: RefetchType, timeout?: number) => void;
+};
+
+const RefetchContext = React.createContext<RefetchContextType>({
+	handleRefetch: () => {},
+});
+
+export const RefetchProvider: React.FC = ({ children }) => {
+	const { useSynthsBalancesQuery } = useSynthetixQueries();
+	const walletAddress = useRecoilValue(walletAddressState);
+
+	const synthsBalancesQuery = useSynthsBalancesQuery(walletAddress);
+	const openOrdersQuery = useGetFuturesOpenOrders();
+	const positionQuery = useGetFuturesPositionForMarket();
+	const positionHistoryQuery = useGetFuturesMarketPositionHistory();
```

Comment:
Left a comment below on the query file but it seems this doesn't set any recoil value inside it and is not being used anywhere else, so maybe a redundant query? Or maybe I'm missing something

Reply:
You're right. It's redundant. I noticed we were querying it for some reason on v2-dev, and wanted to maintain parity, but it seems like it's not being used anywhere else.